Tiltfile

```diff
@@ -10,6 +10,7 @@ localnet_config_path = "localnet_config.yaml"
 localnet_config_defaults = {
     "validator": {"cleanupBeforeEachStart": True},
     "relayminers": {"count": 1},
+    "gateways": {"count": 1},
```

Comment:
Is this for the portal Eric is working on?

Reply:
This is for load testing that will be used to send requests to the `RelayMiners`